Find a functionally equivalent code:android.widget.AbsListView.RecycleBin.addScrapView:COMMENT
Method Modifier: 
Comment:/**
 * Puts a view into the list of scrap views.
 * <p>
 * If the list data hasn't changed or the adapter has stable IDs, views
 * with transient state will be preserved for later retrieval.
 *
 * @param scrap The view to add
 * @param position The view's position within its parent
 */

@@ -11,16 +11,21 @@
         return;
     }
     scrap.dispatchStartTemporaryDetach();
+    // The the accessibility state of the view may change while temporary
+    // detached and we do not allow detached views to fire accessibility
+    // events. So we are announcing that the subtree changed giving a chance
+    // to clients holding on to a view in this subtree to refresh it.
+    notifyViewAccessibilityStateChangedIfNeeded(AccessibilityEvent.CONTENT_CHANGE_TYPE_SUBTREE);
     // Don't scrap views that have transient state.
     final boolean scrapHasTransientState = scrap.hasTransientState();
     if (scrapHasTransientState) {
@@ -39,49 +44,49 @@
         } else {
             mScrapViews[viewType].add(scrap);
         }
-        // Clear any system-managed transient state.
-        if (scrap.isAccessibilityFocused()) {
-            scrap.clearAccessibilityFocus();
-        }
-        scrap.setAccessibilityDelegate(null);
         if (mRecyclerListener != null) {
             mRecyclerListener.onMovedToScrapHeap(scrap);
         }

